TAKING ME
[{u'calltype': u'amd64', u'realsz': 176, u'diff': u'NEW', u'name': u'sym.sig_handler', u'outdegree': 6, u'cc': 1, u'indegree': 1, u'dataxrefs': [], u'nargs': 0, u'codexrefs': [{u'type': u'C', u'addr': 4225369, u'at': 4225232}], u'edges': 0, u'difftype': u'new', u'nlocals': 10, u'cost': 54, u'datarefs': [4239183, 4239194], u'callrefs': [{u'type': u'C', u'addr': 4231392, u'at': 4225279}, {u'type': u'C', u'addr': 4231392, u'at': 4225320}, {u'type': u'C', u'addr': 4231392, u'at': 4225361}, {u'type': u'C', u'addr': 4225376, u'at': 4225369}, {u'type': u'C', u'addr': 4223488, u'at': 4225382}, {u'type': u'C', u'addr': 4223584, u'at': 4225392}], u'offset': 4225232, u'ebbs': 1, u'nbbs': 1, u'type': u'sym', u'size': 176}]
ERR. Target function sym.sig_handler0 does not match sym.sig_handler
Failed to patch all expected patches: 42  total patched: 84
